### PR TITLE
Fixes Magic Bundle

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
# Fixes Magic Bundle

This PR investigates and removes extra bundling from magic that should stay in login.

Fixes https://github.com/EveripediaNetwork/issues/issues/2364

### CODE REVIEW

File: src/components/Elements/Avatar/DisplayAvatar.tsx [hunk 1]
```diff
@@ -48,6 +49,8 @@ const DisplayAvatar = ({
     }
   }, [address, avatarIPFS, setAccount])
 
+  console.log()
```
Comment (kesar) [inline src/components/Elements/Avatar/DisplayAvatar.tsx:52]:
```suggestion
```

File: src/pages/_document.tsx [hunk 2]
```diff
@@ -26,8 +25,6 @@ class MyDocument extends Document<Props> {
             content="black-translucent"
           />
           <link rel="manifest" href="/manifest.json" />
-          <link rel="preconnect" href="https://i.ytimg.com" />
```
Comment (Tolu-Mals) [inline src/pages/_document.tsx:29]: this guy isn't related to magic, its supposed to increase performance on wiki pages by making the video thumbnails load faster. If you can include this only wiki pages, please do.

File: src/pages/_document.tsx [hunk 3]
```diff
@@ -26,8 +25,6 @@ class MyDocument extends Document<Props> {
...
-          <link rel="preconnect" href="https://assets.auth.magic.link" />
```
Comment (Tolu-Mals) [inline src/pages/_document.tsx:30]: Also if you can include this guy only on the login page, it's a performance optimization.

File: src/pages/_document.tsx (hunk 2 above)
Comment (Adebesin-Cell) [reply src/pages/_document.tsx:29]: got it!

File: src/pages/_document.tsx (hunk 3 above)
Comment (Adebesin-Cell) [reply src/pages/_document.tsx:30]: Noted ser

**Review by Royal-lobster — APPROVED** [review]

**Comment by Adebesin-Cell** [comment]
> @Adebesin-Cell what is pending on this pr provide enough reason

There's no pending tasks, it's done.

**Pull request merged**